calcite-qml/Calcite/ToolBar.qml

```diff
@@ -0,0 +1,34 @@
+
+/*******************************************************************************
+ *  Copyright 2012-2022 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+import QtQuick 2.15
+import QtQuick.Templates 2.15 as T
+import QtQuick.Controls 2.15
+import QtQuick.Controls.impl 2.15
+
+T.ToolBar {
+    id: control
+
+    implicitWidth: Math.max(implicitBackgroundWidth + leftInset + rightInset,
+                            contentWidth + leftPadding + rightPadding)
+    implicitHeight: Math.max(implicitBackgroundHeight + topInset + bottomInset,
+                             contentHeight + topPadding + bottomPadding)
+    spacing: 16
+    background: Rectangle {
+        implicitHeight: 48
+        color: "transparent"
+    }
+}
```

Comment:
🔧 One thing to note is that in your current ToolBar implementation, there may be visible gaps between buttons unless they are squished tightly together explicitly with `spacing: 0`:

<img width="227" alt="image" src="https://user-images.githubusercontent.com/42203545/152179249-65a56cae-570a-4b71-93cd-c23a7a58a546.png">

Based on JS FloorFilter / Figma, / Material Style, I  think you should have a filled background, as opposed to being transparent.

Reply:
Fixing this by using the same color used in the `ToolBar` background.
An additional change could be to use by default `spacing : 0` since I cannot see any spacing in the specifications.